File: huxiaotian_parent/huxiaotian_admin_interface/src/main/java/com/yingxiaotian/admin/service/AdminService.java
package com.yingxiaotian.admin.service;

import com.yingxiaotian.pojo.PageResult;
import com.yingxiaotian.pojo.YxtAdmin;
import java.util.List;
public interface AdminService {
    /**
     * 返回全部列表
     * @return
     */
    public List<YxtAdmin> findAll();


    public PageResult findPage(int pageNum, int pageSize);



    /**
     * 增加
     */
    public void add(YxtAdmin user);


    /**
     * 修改
     */
    public void update(YxtAdmin user);


    /**
     * 根据ID获取实体
     * @return
     */
    public YxtAdmin findOne(String username);


    /**
     * 批量删除
     */
    public void delete(String userId);
}

File: huxiaotian_parent/huxiaotian_manager_web/src/main/webapp/js/service/adminService.js
app.service("adminService",function ($http) {
   //添加管理员
   this.add=function (entity) {
      return $http.post("../admin/add.do",entity);
   };
   //查询所有
   this.findAll=function () {
      return $http.get("../admin/findAllNormal.do");
   };
   //分页请求
   this.findPage=function (page,size) {
      return $http.get("../admin/findPage.do?page="+page+"&size="+size);
   };
});